docs/sources/datasources/aws-cloudwatch/aws-authentication/index.md

```diff
@@ -118,3 +145,42 @@ securityContext:
 [configure-grafana-assume-role-enabled]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#assume_role_enabled"
 [configure-grafana-assume-role-enabled]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/setup-grafana/configure-grafana#assume_role_enabled"
 {{% /docs/reference %}}
+
+## Use `Grafana Assume Role`
+
+> Note: Grafan Assume Role is currently in Private Preview for Grafana Cloud. To get early access this feature, reach out to Customer Support and ask for the `awsDatasourcesTempCredentials` feature toggle to be enabled on your account. It is currently only available for CloudWatch.
+
+`Grafana Assume Role` lets Grafana Cloud users authenticate with AWS without having to create and maintain long term AWS Users and rotate their access and secret keys. Instead Grafana Cloud customers can create an IAM Role with permissions to access CloudWatch and a trust relationship with Grafana's AWS Account. Grafana's AWS Account will then make an STS request to AWS to create temporary credentials to access the customer's AWS data. It makes this STS request by passing along an `externalID` that is unique per cloud account, to ensure that Grafana Cloud customers can only access their own AWS data. For more information, refer to the [AWS documentation on external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
+
+To use this `Grafana Assume Role`:
+
+1. Put in a request to Customer Support to enable `awsDatasourcesTempCredentials`
+2. Once the feature is enabled, create a new CloudWatch datasource (or update an existing one) and select `Grafana Assume Role` as an authentication provider
+3. Create a new IAM role in the AWS console, and select Another AWS account as the Trusted entity.
+4. Enter the account ID of the Grafana account that has permission to assume this role 008923505280 and check the Require external ID box.
+5. Enter the external ID specified in the Instructions Box on the CloudWatch Datasource Configuration page in Grafana. This external ID will be unique to your grafana instance.
+6. Attach any required permissions you would like Grafana to be able to access on your behalf (example CloudWatch Logs and CloudWatch Metrics policies)
+7. Give the role a name and description, and click Create role.
+8. Copy the ARN of the role you just created and paste it into the Assume Role ARN field on the CloudWatch Datasource Configuration page in Grafana.
```

Comment:
```suggestion
1. Put in a request to Customer Support to enable`awsDatasourcesTempCredentials`.
2. Once the feature is enabled, create a new CloudWatch data source (or update an existing one) and select `Grafana Assume Role` as an authentication provider.
3. Create a new IAM role in the AWS console, and select **Another AWS account** as the Trusted entity.
4. Enter the account ID of the Grafana account that has permission to assume this role 008923505280 and check the **Require external ID** box.
5. Enter the external ID specified in the instructions box on the **Settings** tab of CloudWatch data source configuration in Grafana. This external ID will be unique to your Grafana instance.
6. Attach any required permissions you would like Grafana to be able to access on your behalf (for example, CloudWatch Logs and CloudWatch Metrics policies).
7. Give the role a name and description, and click **Create role**.
8. Copy the ARN of the role you just created and paste it into the **Assume Role ARN** field on the **Settings** tab of CloudWatch data source configuration in Grafana.
```

Reply:
Some questions:

1. In step 3 is Trusted entity a field name?
2. What's the number in step 4?
3. On step 5, is this a titled part of the UI or just a place where there are instructions?